clang/lib/CodeGen/CodeGenModule.cpp

```diff
@@ -7149,7 +7149,6 @@ void CodeGenModule::EmitTopLevelDecl(Decl *D) {
   case Decl::ObjCCompatibleAlias:
     ObjCRuntime->RegisterAlias(cast<ObjCCompatibleAliasDecl>(D));
     break;
-
```

Comment:
We don't change files outside CIR dir without a good reason, formatting is specially bad for rebasing, so you need to remove the changes to this file.